components/progressbar/index.css

```diff
@@ -10,51 +10,86 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-progressbar.css";
-@import "../vars/css/components/spectrum-meter.css";
-@import "../vars/css/components/spectrum-fieldlabel.css";
+.spectrum-ProgressBar {
+  /* Static tokens */
+  --spectrum-progressbar-animation-ease-in-out-indeterminate: cubic-bezier(.45, 0, .40, 1);
+  --spectrum-progressbar-animation-duration-indeterminate: 1000ms;
+  --spectrum-progressbar-corner-radius: var(--spectrum-corner-radius-100);
+
+  --spectrum-progressbar-fill-size-indeterminate: 70%;
+  /* --spectrum-global-dimension-static-size-2400 */
+  --spectrum-progressbar-size-2400: 192px;
+  /* --spectrum-global-dimension-static-size-2500 */
+  --spectrum-progressbar-size-2500: 200px;
+  /* --spectrum-global-dimension-static-size-2800 */
+  --spectrum-progressbar-size-2800: 224px;
```

Comment:
These were not in the XD file and I didn't find any tokens that made sense in spectrum-tokens. Is there a better approach? 

For the progressbar size, I referenced https://spectrum.adobe.com/page/progress-bar/#Width and https://github.com/adobe/spectrum-css/blob/main/components/expressvars/css/components/spectrum-progressbar.css.